moabb/evaluations/evaluations.py

```diff
@@ -461,60 +587,117 @@ def evaluate(self, dataset, pipelines):
         run_pipes = {}
         for subject in dataset.subject_list:
             run_pipes.update(self.results.not_yet_computed(pipelines, dataset, subject))
-        if len(run_pipes) == 0:
-            return
+        if len(run_pipes) != 0:
```

Comment:
Why change this test, it removes a level of indentation and improve readability of the code and it will mess with the git blame. If you have no specific reason to change this test, please revert.